src/main/kotlin/app/revanced/patches/youtube/video/speed/custom/CustomPlaybackSpeedPatch.kt

```diff
@@ -30,17 +28,23 @@ import com.android.tools.smali.dexlib2.iface.reference.FieldReference
 import com.android.tools.smali.dexlib2.iface.reference.MethodReference
 import com.android.tools.smali.dexlib2.immutable.ImmutableField
 
-@Name("Custom playback speed")
-@Description("Adds custom playback speed options.")
-@DependsOn([IntegrationsPatch::class, LithoFilterPatch::class, SettingsPatch::class, BottomSheetHookPatch::class])
-class CustomPlaybackSpeedPatch : BytecodePatch(
-    listOf(
+@Patch(
+    description = "Adds custom playback speed options.",
+    dependencies = [IntegrationsPatch::class, LithoFilterPatch::class, SettingsPatch::class, BottomSheetHookPatch::class]
+)
```

Comment:
```suggestion
@Patch(
    name = "Custom playback speed",
    description = "Adds custom playback speed options.",
    dependencies = [IntegrationsPatch::class, LithoFilterPatch::class, SettingsPatch::class, BottomSheetHookPatch::class]
)
```

Reply:
This patch is a dependency of `PlaybackSpeedPatch`